<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="7">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="comboBox_options"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Refresh">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QComboBox" name="comboBox_list_boot"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Include explanations</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="nextsearch">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="prevsearch">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_Final">
      <property name="text">
       <string>Show Output</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
